### PR TITLE
Chunk customer set using row range parameter.

### PR DESCRIPTION
**Pull request by wilsjame**
Bare bones row range functionality with parameter validation in the controller. 

Example queries:

http://localhost/searchspring/boost/customers?dateRange=All&rowRange=1,250
http://localhost/searchspring/boost/customers?dateRange=All&rowRange=251,500

If the rowRange is out of bounds, like extending beyond the last element in the customer set, no errors are thrown and the return chunk includes any elements still in the range. 

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202765172217467

### CODE REVIEW

**Comment by wilsjame** [comment]
@Ipswich Make any changes you want 🙏 I worked off update JSON response format #15 so if this gets merged we can scrap that pr.

File: Helper/Customer.php [hunk 1]
```diff
@@ -71,14 +73,48 @@ public function validateDateRange()
...
+                    $isValidRowRange = false;
+            }
+            if (isset($result[1])){
+                $result[1] = (int) $result[1];
+                if ($result[1] <= 0 )
+                    $isValidRowRange = false;
+            }
+        }
+        return $isValidRowRange;
+    }
```
Comment (Ipswich) [inline Helper/Customer.php:110]: Things get a little weird if you do something like 100,5 for rowRange. Parameters like that produce [100, 101, 102, 103, 104]. Maybe not bad weird, but might be undesired behavior we should filter.

File: Helper/Customer.php [hunk 2]
```diff
@@ -101,23 +137,23 @@ public function getCustomers()
             $customerCollection->getSelect()->where("($whereCreatedAt) || ($whereUpdatedAt)"); // Query string
         }
 
+        // Chunk customers with row range.
+        $rowRange = $this->getRowRange();
+        if (isset($rowRange[0]) && isset($rowRange[1])) 
+            $customerCollection->getSelect()->limit((int)$rowRange[1], (int)$rowRange[0]);
```
Comment (Ipswich) [inline Helper/Customer.php:143]: Do you need to recast rowRange as an int? It looks like that gets done at the end of getRowRange.

**Review by Ipswich — CHANGES_REQUESTED** [review]
I dig it, looks a lot cleaner, definitely an improvement haha.

File: Helper/Customer.php (hunk 1 above)
Comment (Ipswich) [reply Helper/Customer.php:110]: ...Unless I'm misreading the intent. I just looked up the limit function. I was thinking row range would be like:
`rowRange=5,10` returns rows 5 through 10.
Instead it behaves like:
`rowRange=5,10` gets 10 rows from element 5 onwards.

Either is totally acceptable I think, but it should probably be explained somewhere to reduce confusion. Although I might be the only confused one 😬

File: Helper/Customer.php (hunk 2 above)
Comment (Ipswich) [reply Helper/Customer.php:143]: ...Just looked at the fourtell code, it's tried and true over there, this is probably an unnecessary comment lol.

**Review by Ipswich — APPROVED** [review]

**Pull request merged**